### PR TITLE
Support MinGW

### PR DESCRIPTION
**Pull request by ashtonmeuser**
Compiling using MinGW requires `.a` library not `.dll`. Also see https://github.com/WebAssembly/wasm-c-api/pull/182.

### CODE REVIEW

**Comment by syrusakbary** [comment]
This makes 100% sense, let's wait until merged upstream :)

**Comment by ashtonmeuser** [comment]
Merged in https://github.com/WebAssembly/wasm-c-api/commit/13f3bfd5280777c4b00848315617527728a294b2.

**Review by ptitSeb — APPROVED** [review]

**Pull request merged**
